python/ray/tests/test_object_spilling.py

```diff
@@ -295,6 +295,51 @@ def test_unstable_spill_objects_automatically(unstable_spilling_config,
     assert_no_thrashing(address["redis_address"])
 
 
+@pytest.mark.skipif(
+    platform.system() == "Windows", reason="Failing on Windows.")
+def test_slow_spill_objects_automatically(object_spilling_config,
+                                          shutdown_only):
+    # Limit our object store to 75 MiB of memory.
+    # object_spilling_config, _ = slow_spilling_config
+    object_spilling_config, _ = object_spilling_config
+    address = ray.init(
+        num_cpus=1,
+        object_store_memory=75 * 1024 * 1024,
+        _system_config={
+            "max_io_workers": 4,
+            "automatic_object_spilling_enabled": True,
+            "object_store_full_delay_ms": 100,
+            "object_spilling_config": object_spilling_config,
+            "min_spilling_size": 0
+        })
+    replay_buffer = []
+    solution_buffer = []
+    buffer_length = 10
+
+    # Create objects of more than 800 MiB.
+    # Each object is {32, 40, 48} MiB.
+    for _ in range(buffer_length):
+        ref = None
+        while ref is None:
+            multiplier = random.choice([4, 5, 6])
```

Comment:
@stephanie-wang I wrote a test to emulate a "slow filesystem" where each spilling action will take 1 - 2 seconds, thus testing the OOM limit. This test works with the default object sizes (8, 16, 24 MiB).

However when I changed this line to `[4,5,6]`, i.e. object sizes are `[32, 40, 48]` MiB, all `ray.get` calls fail except for the first one, with "Object Store Full" error. This occurs even with the default object spilling config (not slow fs). Do you think this indicates some other problem in the spilling/restoring code? I suspect this is similar to the case with `AllocateReturnObjects`, but is not fixed by that PR.

Reply:
This was probably #15886, right? I think it's probably okay!